modmain.lua

```diff
@@ -2,21 +2,41 @@ local require = GLOBAL.require
 
 require "simutil"
 require "constants"
+-- require "prefabs/player_common" -- do we need this ???
 
 local ImageButton = require "widgets/imagebutton"
 local RecipePopup = require "widgets/recipepopup"
 local StickyRecipePopup = require "widgets/stickyrecipepopup"
 local RecipePopup_Refresh_base = RecipePopup.Refresh or function() return "" end
 local STICKYRECIPE_OptionPos = GetModConfigData("StickyRecipePopup_AltPos") or "original"
 
+local function onload(inst, data) -- on preload or onload ???
+	if data then
+		if data.stickiedrecipe then
+			GLOBAL.GetPlayer().stickiedrecipe = data.stickiedrecipe --get back saved data
+			GLOBAL.GetPlayer().HUD.controls.stickyrecipepopup:Refresh() --call a refresh of the screen ???

```

Comment:
I'm not sure this is what I should call to provoke a display of the stickyrecipe popup